synapse/http/site.py

```diff
@@ -59,8 +61,9 @@ class SynapseRequest(Request):
         logcontext: the log context for this request
     """
 
-    def __init__(self, channel, *args, **kw):
+    def __init__(self, channel, *args, max_request_body_size=1024, **kw):
```

Comment:
This seems like a surprisingly low default? Can we not make it a required field?

Reply:
the default was mostly to avoid messing around with the tests more than I had to. Happy to make it required if you prefer.